force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -34,18 +34,26 @@ public with sharing class DummyJSONCallout {
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
 
-	public static void getDummyJSONUserFromId(String dummyUserId) {
+	@future(callout=true)
+	 public static void getDummyJSONUserFromId(String dummyUserId) {
 		// Create HTTP request to send.
+		HttpRequest req = new HttpRequest();
 
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+		req.setEndpoint('https://dummyjson.com/user/' + dummyUserId);
```

Comment:
Best practice would be to use Named Credentials. Make sure you know how to do this for the capstone.